index.html

```diff
@@ -128,101 +128,46 @@ <h3>Multi-Post Stories</h3>
-              sign ups required ha been the industry standups
-            </p>
-          </div>
-          <ul class="grid-container2">
-            <li class="item1"><a href="#">html</a></li>
-            <li class="item2"><a href="#">bootstrap</a></li>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
-        <div class="job5">
-          <h2>Profesional Art Printing Data</h2>
-          <div class="info-data">
-            <p>
-              A daily section of daily private personalised reads; no accounts
-              or sign ups required ha been the industry standups
-            </p>
-          </div>
-          <ul class="grid-container2">
-            <li class="item1"><a href="#">html</a></li>
-            <li class="item2"><a href="#">bootstrap</a></li>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
-        <div class="job6">
-          <h2>Profesional Art Printing Data</h2>
-          <div class="info-data">
-            <p>
-              A daily section of daily private personalised reads; no accounts
-              or sign ups required ha been the industry standups
-            </p>
-          </div>
-          <ul class="grid-container2">
-            <li class="item1"><a href="#">html</a></li>
-            <li class="item2"><a href="#">bootstrap</a></li>
-            <li class="item3"><a href="#">Ruby</a></li>
-          </ul>
-          <button type="button" class="job-button">See Project</button>
-        </div>
+      <!--modal-->
+      <div class="modal">
+        <div class="poster poster-modal">
+         <div class="post-information modal-info">
+         <div class="post-title modal-posttitle">
+          <h3 class="heading-modal">Multi-Post Stories</h3>
+          <div class="close-modal">&times;</div>
+         </div>
+         <ul class="grid-container" id="modal-list-grid">
+         
+         </ul>
+         <div class="image-holder image-modal">
+          <img class="post-images" src="./assets/post2.svg" alt="Post" />
+         </div>
+        </div>   
+     <div class="post post-modal">
+      <p class="paragraph"> 
+        A daily selection of private personalised reads; o accounts or
+            signups required. Has been the industry's dummy standard text ever
+            snce the 1500s. When an unknown printer took a standard dummy text
+      </p>
+    </div>
+     <div class="btns">
+      <div class="btns-container">
+      <span class="work-btns">See Live</span>
+      <img class="icons" src="./assets/github.png" alt="">
+    </div>
+    <div class="btns-container">
+      <span class="work-btns">See Live</span>
+      <img class="icons" src="./assets/backto.png" alt=""/>
+    </div>
+     </div>
```

Comment:
- [ ] Kindly make sure that those elements are buttons, instead of a `div` with a `span` inside it. Also, kindly add the button interactions (hover, active, ...etc).